lib/Widget/postHosTable.dart

```diff
@@ -105,9 +117,11 @@ class _PostHosTableState extends State<PostHosTable> {
       "age": "22 ปี",
       "room": "3",
       "bed": "2",
-      "o2": "99%",
-      "p": "122",
       "t": "37.8",
+      "r": "26",
+      "hr": "122",
+      "bp": "90",
+      "o2": "99%",
```

Comment:
Rename to meaningful name.  For ex. t = >temperature, r=> respirationRate sth...